engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -30,7 +30,7 @@ <h4 translate> Basic Information </h4>
                   type="text"/>
       </div>
 
-      <div bst-form-group label="{{ 'Type' | translate }}">
+      <div bst-form-group label="{{ 'Type (Puppet and ostree will no longer be supported in Katello 3.16.)' | translate }}">
```

Comment:
I think the messaging is a good idea, but it might be better to add this as a separate block of text that way the translation for 'Type' won't be broken